scripts/generate_code.py

```diff
@@ -407,6 +407,13 @@ def glob(path, pattern):
 
 flatc(["--csharp", "--gen-object-api"], schema=type_field_collsion_schema)
 
+# Union / value collision
+flatc(
+    BASE_OPTS + CPP_OPTS + CS_OPTS + ["--gen-onefile"],
```

Comment:
I think you just need CS_OPTS at the moment? You just made a change to the c# code gen and this `union-value_collection.fbs` is new, so there shouldn't be any other generated code associated with this.

Reply:
--gen-object-api is also required (--cs-gen-json-serializer requires --gen-object-api to be set as well.).

Reply:
Could you just explicitly list those out here then? If you include `CPP_OPTS` that includes the `--cpp` flag and will generate the C++ code, which i don't think we need at the moment.

Reply:
@dbaileychess 

I already removed them and the generated C++ Source filed:

![image](https://user-images.githubusercontent.com/32997632/211168950-c9be0c9c-4b0c-4d43-904a-a6ec808e7b45.png)

Reply:
Ah, sorry didn't check.